app/src/main/assets/chart.html

```diff
@@ -7,6 +7,18 @@
 <script src="jquery-1.5.1.min.js"></script>
 <script src="freezeheader.js"></script>
 <script src="chart.js"></script>
+<script>
+  function isRowEmpty(parentNode) {
```

Comment:
@capnfabs I tried but pebble worn me out. There is a bug regarding variable scope in pebble and that's why I choose this solution. We really need to revome pebble as soon as possible.